sensor_msgs/PointCloud2[] pick_cluster
---
bool done